home/templatetags/survey.py

```diff
@@ -21,3 +21,33 @@ def create_star(number, id_element, num_stars):
     return utils_create_star(
         active_star=int(number), num_stars=num_stars, id_element=id_element
     )
+
+
+@register.simple_tag(takes_context=True)
+def pagination_url(context, page_number):
```

Comment:
Can this be replaced with https://docs.djangoproject.com/en/5.2/ref/templates/builtins/#querystring ?